{"Message":"由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 10.80.52.218:20702","SocketErrorCode":10060,"ErrorCode":10060,"NativeErrorCode":10060,"StackTrace":"   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)\r\n   at System.Net.Sockets.Socket.Connect(EndPoint remoteEP)\r\n   at System.Net.Sockets.Socket.Connect(IPAddress address, Int32 port)\r\n   at System.Net.Sockets.TcpClient.Connect(String hostname, Int32 port)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Net.Sockets.TcpClient.Connect(String hostname, Int32 port)\r\n   at System.Net.Sockets.TcpClient..ctor(String hostname, Int32 port)\r\n   at CaseManagerCore.Population.NextYear() in F:\\MyProject\\Cpp\\ChinesePopulationPrediction\\Server\\CaseManagerCore\\Population.cs:line 590\r\n   at CoreWsApp2.Startup.Echo(WebSocket webSocket) in F:\\MyProject\\Cpp\\ChinesePopulationPrediction\\Server\\CoreWsApp2\\Startup.cs:line 693","Data":{},"InnerException":null,"HelpLink":null,"Source":"System.Net.Sockets","HResult":-2147467259}